sklearn/tests/test_grid_search.py

```diff
@@ -220,19 +220,10 @@ def test_grid_search_score_method():
     # Check warning only occurs in situation where behavior changed:
```

Comment:
There was `score_no_scoring` variable that is being tested for not raising a warning. And, `score_no_score_auc` variable also tests for no warning being raised. But I am not sure what to do with the latter, ( I think pep8 will fail there)